### PR TITLE
Updated with latest phpcs ruleset.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
updated with the latest phpcs ruleset.

### CODE REVIEW

File: classes/class-bsf-custom-fonts-render.php [hunk 1]
```diff
@@ -162,7 +164,7 @@ public function add_style() {
 				}
 				?>
 				<style type="text/css">
-					<?php echo strip_tags( $this->font_css ); ?>
+					<?php echo wp_kses_post( wp_strip_all_tags( $this->font_css ) ); ?>
```
Comment (premanshup) [inline classes/class-bsf-custom-fonts-render.php]: `wp_kses_post` - we can replace this with `esc_html()` ?

File: classes/class-bsf-custom-fonts-render.php [hunk 2]
```diff
@@ -239,7 +241,7 @@ private function render_font_css( $font ) {
 					$arr[] = 'url(' . esc_url( $links['font_ttf'] ) . ") format('truetype')";
 				}
 				if ( $links['font_otf'] ) {
-					$arr[] = 'url(' . esc_url( $links['font_otf'] ) . ") format('opentype')";
+					$arr[] = 'url(' . $links['font_otf'] . ") format('opentype')";
```
Comment (premanshup) [inline classes/class-bsf-custom-fonts-render.php]: No need to remove `esc_url` from here.

**Review by premanshup — CHANGES_REQUESTED** [review]

File: classes/class-bsf-custom-fonts-render.php (hunk 1 above)
Comment (shubhw12) [reply classes/class-bsf-custom-fonts-render.php]: Done

**Review by premanshup — APPROVED** [review]

File: includes/class-bsf-custom-fonts-admin.php [hunk 3]
```diff
@@ -93,7 +95,7 @@ public function custom_fonts_menu_highlight() {
 			global $parent_file, $submenu_file;
```
Comment (Balachandark) [inline includes/class-bsf-custom-fonts-admin.php]: Why are we using $parent_file global variable?

**Review by Balachandark — CHANGES_REQUESTED** [review]

**Review by UttamSharma007 — APPROVED** [review]

**Review by premanshup — APPROVED** [review]

File: includes/class-bsf-custom-fonts-admin.php (hunk 3 above)
Comment (shubhw12) [reply includes/class-bsf-custom-fonts-admin.php]: It is compulsory to override the global variable

**Review by Balachandark — APPROVED** [review]

**Pull request merged**